steps/check-your-appeal/CheckYourAppeal.js

```diff
@@ -19,6 +19,7 @@ class CheckYourAppeal extends CYA {
 
         return [
             section('benefit-type',                     { title: this.content.benefitType }),
+            section('have-a-mrn',                       { title: this.content.compliance.haveAMRN }),
```

Comment:
Looking at https://track-your-appeal.herokuapp.com/prototype-beta-23/submit-your-appeal/017-check-your-answers there is no requirement to add a new section. This is not required.